pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.javaslang</groupId>
+    <artifactId>javaslang-docs</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
I agree if we sync version number with `javaslang` project. 
How about if docs grow faster/slower than `javaslang` project i.e. when `javaslang` 2.0.0 released but docs only cover 50% of the API?